### ISSUE TITLE
Test cases failing with not implemented: service-requests.cpp:259 extract_spoff

### ISSUE THREAD

**Issue by racardoso** [PPC64/hhvm#121]
Some test cases fails on quick test suite due a not implemented:

```
--hphp/runtime/vm/jit/service-request.cpp:259
 FPInvOffset extract_spoff(TCA stub) { case Arch::PPC64: not_implemented() }
```

This happens because `DecodedInstruction` (and `isLea()` function) has not yet implemented for ppc64
- [ ] Implement `isLea()`
- [ ] Implement `case Arch::PPC64: not_implemented()`

The following tests failing due this error:

cat quick/debugger/break1.php.diff
cat quick/debugger/flow.php.diff
cat quick/debugger/flow2.php.diff
cat quick/debugger/flow_break_interference.php.diff
cat quick/debugger/flow_excep.php.diff
cat quick/debugger/flow_gen.php.diff
cat quick/debugger/flow_gen_async.php.diff
cat quick/debugger/flow_gen_async_excep.php.diff
cat quick/debugger/flow_gen_excep.php.diff
cat quick/debugger/flow_multistep.php.diff
cat quick/debugger/flow_small.php.diff
cat quick/debugger/stack1.php.diff

**Comment by racardoso** [comment]
related to issue #24

**Comment by lbianc** [comment]
Only 3 tests were fixed with the change (0145f1a109671bd38df97a5c38ebd1a49b5d5b11). The others have an assertion false on the stack, what can be related with the offset returned from the decoder (a new task will be created for that).
As it is failing in other part of the code, I'm closing this task.

Tests passing after change:

hphp/test/quick/debugger/break1.php
hphp/test/quick/debugger/flow2.php
cat quick/debugger/flow_gen.php
